### ISSUE TITLE
allow printing class names with space instead of bullet point

### ISSUE THREAD

**Issue by 190n** [schoero/eslint-plugin-better-tailwindcss#275]
### Description

I noticed that when class names are printed, spaces are converted to bullet points:

https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/a9876d0f7660b3c0fe02eaa56050bb13ccac861d/src/utils/utils.ts#L39-L45

```
│   108:21  error  Incorrect class order. Expected
│ 
│ "text-2xl·font-bold·leading-[34px]·text-text"
│ 
│ to be
│ 
│ "text-2xl·leading-[34px]·font-bold·text-text"  better-tailwindcss/enforce-consistent-class-order
```

I don't love this behavior because if I simply copy the class name that is printed, my code won't work but the lint error will go away. It makes it harder to manually apply lint suggestions from the output. Would it be possible to make this behavior configurable?

**Comment by schoero** [comment]
This is now configurable via the [`messageStyle`](https://github.com/schoero/eslint-plugin-better-tailwindcss/blob/main/docs/settings/settings.md#messagestyle) option in the settings object.

```ts
settings: {
  "better-tailwindcss": {
    messageStyle: "raw"
  }
}
```
